src/roles/harlot.py

```diff
@@ -8,69 +8,66 @@
 import src.settings as var
 from src.utilities import *
 from src import channels, users, debuglog, errlog, plog
-from src.functions import get_players, get_all_players, get_main_role
-from src.decorators import cmd, event_listener
+from src.functions import get_players, get_all_players, get_main_role, get_target
+from src.decorators import command, event_listener
 from src.messages import messages
 from src.events import Event
 
 VISITED = {}
+PASSED = set()
```

Comment:
It'd be nice if we added type annotation comments to VISITED and PASSED. Even though we aren't using mypy (yet), it's still helpful when browsing source to see what sorts of structure things are at a glance.